src/lighteval/models/base_model.py

```diff
@@ -373,7 +373,11 @@ def greedy_until_multi_turn(  # noqa: C901
             dataloader, desc="Greedy Multi Turn generation", position=1, leave=False, disable=self.disable_tqdm
         ):
             request = request_batch[0]
-            stop_tokens = request.stop_sequence
+            # For chat models, generation stops with EOS token, so we don't need to specify stop tokens
+            if hasattr(self.tokenizer, "chat_template") and self.tokenizer.chat_template is not None:
```

Comment:
Does this work if I run a model that has a chat template available but i'm not launching lighteval with  `--use_chat_template` ?